### PR TITLE
DYN-5227-NotificationCenter-Font

### PR DESCRIPTION
**Pull request by filipeotero**
Importing ArtifaktElement from @hig/fonts

image

FYI:

@avidit @QilongTang

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -25,7 +25,6 @@ module.exports = (env) => {
         },
         {
           test: /\.(scss|css)$/,
-          exclude: /node_modules/,
```
Comment (QilongTang) [inline webpack.config.js:28]: This is for?

**Review by QilongTang — APPROVED** [review]
LGTM with one question

File: webpack.config.js (hunk 1 above)
Comment (filipeotero) [reply webpack.config.js:28]: By excluding the node_modules folder, webpack doesn't transform the files included in that folder. I imported the `ArtifaktElement.css` file into the project, which is in the node_modules folder. I removed that line so webpack could include the `ArtifaktElement.css.

**Comment by filipeotero** [comment]
@RobertGlobant20 @zeusongit @jesusalvino Could you also take a look, please? Thanks!

**Review by RobertGlobant20 — APPROVED** [review]
LGTM

**Review by zeusongit — APPROVED** [review]

**Pull request merged**